Tasks/Maven/pmdformaven.ts

```diff
@@ -10,25 +10,27 @@ import trm = require('vsts-task-lib/toolrunner');
 
 import ar = require('./analysisresult');
 
+export const toolName:string = 'PMD';
+
 // Adds PMD goals, if selected by the user
 export function applyPmdArgs(mvnRun: trm.ToolRunner):void {
-    mvnRun.arg(['jxr:jxr', 'pmd:pmd']); // JXR source code cross-referencing is a required pre-goal for PMD
```

Comment:
Hhhmm. That's a bit trickier. I'd suggest omitting it for now and Jean-Marc can decide whether the user experience is acceptable.